src/components/welcome-message-prompt.tsx

```diff
@@ -48,21 +49,18 @@ export const ExampleMessagePrompt = ( {
 	className,
 }: ExampleMessagePromptProps ) => (
 	<div className={ cx( 'flex mt-2' ) }>
-		<div
```

Comment:
@katinthehatsite to ensure I do not overlook something, I wanted to ask: Was there a specific reason to leverage a `div` rather than `button` in https://github.com/Automattic/studio/pull/245 for this interactive element? 

For context, it is generally a discouraged practice as it is easy to overlook the numerous accessibility features built into default interactive elements like `button` and `a`. 

Thank you in advance for sharing any insight you may have! 🙇🏻

Reply:
Hi @dcalhoun - thanks for the ping! It was mostly a leftover on my end, I think. I believe I started the implementation of example prompts as non-interactive elements and when switching to interactive elements did not adjust it afterwards. 

Thanks for making the change! It is definitely something for me to be aware of in the future implementation 🙇

Reply:
> [...] do you have some specific references/checklist that you personally use to ensure that elements that you are implementing include all the necessary accessibility features?

@katinthehatsite good question! 

At a high level, I think it is really important to practice exploring the product from a user's perspective. E.g., navigate the product as a user whose limited motor skills inhibit cursor usage and therefore instead relies upon a keyboard or navigate the product as a user with limited sight (I close my eyes!) and is dependent upon a screen reader. Doing so will likely result in encountering confusing or challenging experiences — a frustrating focus reset that forces you to regain your bearings, an interactive element that is entirely inaccessible, etc. 

In regards to specific techniques or tools, here a few that come to mind: 

* Accessible UI is difficult to implement _afterwards_, it is something that should be considered during the design phase and requires collaboration with designers. 
* As mentioned in my opening statement, always navigate the product with input mechanisms (cursor, keyboard, screen reader, etc) that are not your default. Even try combining input mechanisms. 
* Consider and design accessibility at the component level and avoid one-off implementations. E.g., collaborate with designers to build an accessible `Button` component that includes all the necessary states, attributes, etc; avoid using `button` (or `div`) directly when it's possible to use the thoughtfully-designed component. 
* When building UI, spend time comparing your implementation (from a user's perspective) against similar experiences built by high-profile, highly-regarded companies (e.g., Apple, Google, Facebook, Amazon). 
* Regarding web accessibility best practices, the free [axe DevTools](https://www.deque.com/axe/browser-extensions/) Chrome Extension can help catch rudimentary issues. It is a bit less helpful/relevant in the context of Electron, but it is still possible to [use the extension](https://www.electronjs.org/docs/latest/tutorial/devtools-extension#manually-loading-a-devtools-extension) with some benefit. Keep in mind that while it will catch rudimentary issues, it cannot replace thoughtful accessibility design. 
* When building new features, it might be most effective to first build functionality (use a `button` not a `div`), the implement the visual design (achieve the desired styling), and finally retest functionality (ensure styling approach didn't produce regressions). 
* Consider subscribing to [A11y Weekly](https://a11yweekly.com) as a continuing education source for thought pieces on more complex accessibility design and practices. 

Overall, there are some fundamental aspects of accessible design that are enforceable as ["rules"](https://www.w3.org/WAI/standards-guidelines/aria/) (we might consider enforcing the ones we can with automated [tests](https://github.com/nickcolley/jest-axe) or [linter](https://github.com/dequelabs/axe-core-npm/tree/develop/packages/react) [config](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y) where possible), but the vast majority of it is subjective and requires a fair amount of thought, placing oneself in the user's "shoes."  

Hope this helps! 🙇🏻